README.md

```diff
@@ -1 +1,67 @@
-# Webpack
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-do list project.
+
+> This is the javaScript project for Microverse. In this project, I built a simple to-do list useing HTML, CSS, javaScript and WebPack.
```

Comment:
- [ ] Kindly update the name of the project and add a little description about it.